code/game/jobs/jobs.dm

```diff
@@ -1,3 +1,17 @@
+//delete this after adaptation
+
+var/const/ENG               =(1<<0)
+var/const/SEC               =(1<<1)
+var/const/MED               =(1<<2)
+var/const/SCI               =(1<<3)
+var/const/CIV               =(1<<4)
+var/const/COM               =(1<<5)
+var/const/MSC               =(1<<6)
+var/const/SRV               =(1<<7)
+var/const/SUP               =(1<<8)
+var/const/SPT               =(1<<9)
+
```

Comment:
Why you add that defines?
Do you know that you break crew manifest?
Them conflict with existence one. When you add them did you check where them used?

Security in Engineering category
![111](https://user-images.githubusercontent.com/2458675/47619288-e565c300-dafe-11e8-94f3-2ea8be402050.png)

Reply:
Its known issue to me, we had a discussion with Nanako and we agreed that fix for this will come with pref port.
Current preferences is a mess that contains things like (var/const/MEDSCI and var/const/ENGSEC) and not suitable for this programm

Reply:
why is engsec even a thing? Who did that

Reply:
@NanakoAC 
That var previously used only for savefiles. Can't say for sure why they merge eng and sec, but probably to reduce prefs dropping when jobs list changes.

Reply:
@LagannPM 
>Its known issue to me, we had a discussion with Nanako and we agreed that fix for this will come with pref port.

And leave that part broken? Why not remove one group of constants and fix all?

Reply:
Yes, its not a major bug. Fixing this means rework of prefs, which was planned anyway. If only this PR was merged prefs port would be ready already